### PR TITLE
100k BAL Acquisition

### PR DESCRIPTION
**Pull request by 0xrajath**
## Motivation
This includes an AIP for the 100k BAL Acquisition as part of Strategic Partnership with Balancer DAO Part 2
More details can be found on the governance forum: https://governance.aave.com/t/arc-strategic-partnership-with-balancer-part-2/7813

### CODE REVIEW

File: content/aips/58D30E766E47-AIP-100K-BAL-ACQUISITION.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: Strategic Partnership with Balancer DAO Part 2 - 100k BAL Acquisition
+status: Proposed
```
Comment (sakulstra) [inline content/aips/58D30E766E47-AIP-100K-BAL-ACQUISITION.md:3]: you no longer need to supply status

File: content/aips/58D30E766E47-AIP-100K-BAL-ACQUISITION.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
```
Comment (sakulstra) [inline content/aips/58D30E766E47-AIP-100K-BAL-ACQUISITION.md:1]: the proposal no longer needs the `<hash>-AIP` prefix.
`100K-BAL-ACQUISITION.md` is fine.

File: content/aips/58D30E766E47-AIP-100K-BAL-ACQUISITION.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+The aUSDC does not leave the Aave V2 Collector as the Bonding Curve enables the aUSDC to be transferred from the Collector to the trader's wallet upon the Collector receiving BAL from the trader.
+
+The Proposal Payload does the following:
+1. Approve the Bonding Curve contract to spend 800,000 aUSDC from the Aave V2 Collector.
+
+The Bonding Curve contract has the following functions and public storage variables:
+1. `function purchase(uint256 amountIn, bool toUnderlying) external returns (uint256)`: This function allows the trader to purchase aUSDC/USDC for BAL, with the trader receiving a 50 bps reward over the BAL/USD Chainlink oracle price. It takes ` uint256 amountIn` (BAL amount input including 18 decimals) and `bool toUnderlying` (Whether to receive as USDC (true) or aUSDC (false)) as input parameters and returns the amount of aUSDC/USDC (including 6 decimals) received on purchase.
```
Comment (sakulstra) [inline content/aips/58D30E766E47-AIP-100K-BAL-ACQUISITION.md:44]:
```suggestion
1. `function purchase(uint256 amountIn, bool toUnderlying) external returns (uint256)`: This function allows the trader to purchase aUSDC/USDC for BAL, with the trader receiving a 50 bps reward over the BAL/USD Chainlink oracle price. It takes `uint256 amountIn` (BAL amount input including 18 decimals) and `bool toUnderlying` (Whether to receive as USDC (true) or aUSDC (false)) as input parameters and returns the amount of aUSDC/USDC (including 6 decimals) received on purchase. 
```

**Comment by sakulstra** [comment]
also the target is wrong - should be `main` now :sweat_smile:

File: content/aips/58D30E766E47-AIP-100K-BAL-ACQUISITION.md (hunk 1 above)
Comment (0xrajath) [reply content/aips/58D30E766E47-AIP-100K-BAL-ACQUISITION.md]: Done

File: content/aips/58D30E766E47-AIP-100K-BAL-ACQUISITION.md (hunk 3 above)
Comment (0xrajath) [reply content/aips/58D30E766E47-AIP-100K-BAL-ACQUISITION.md]: Done

**Pull request merged**